OmgvaPOS/src/ReservationManagement/Mappers/ReservationMapper.cs

```diff
@@ -0,0 +1,56 @@
+﻿using OmgvaPOS.ReservationManagement.DTOs;
+using OmgvaPOS.ReservationManagement.Enums;
+using OmgvaPOS.ReservationManagement.Models;
+
+namespace OmgvaPOS.ReservationManagement.Mappers
+{
+    public static class ReservationMapper
+    {
+        public static ReservationDto ToDto(Reservation reservation)
```

Comment:
You can do a funny thing here. If you add "this" before "Reservation" this method would become an exttension method. Then, instead of calling "var resDTO = ReservationMapper.ToDto(reservation)", you could just do  "var resDTO = reservation.ToDto()". In my opinion. It looks cleaner.

Reply:
Same with other medthods.